include/dosbox.h

```diff
@@ -55,15 +55,6 @@ class Config;
 using config_ptr_t = std::unique_ptr<Config>;
 extern config_ptr_t control;
 
-enum MachineType {
-	MCH_HERC,
-	MCH_CGA,
-	MCH_TANDY,
-	MCH_PCJR,
-	MCH_EGA,
-	MCH_VGA
-};
-
```

Comment:
I Don't think this change belongs in this PR?

Reply:
The machine type is DOSBox's fundamental differentiation, and this makes it bit-ops capable. So I lumped it into the overall bit-ops improvement.